tests/model_hub_tests/torch_tests/requirements_secondary.txt

```diff
@@ -1,4 +1,5 @@
 -c ../../constraints.txt
 # This file contains requirements dependednt from modules in requirements.txt
-git+https://github.com/facebookresearch/detectron2.git 
+# get immutable commit to be able to use pip cache
+git+https://github.com/facebookresearch/detectron2.git@1a4df4d954089534daebbd7a0a45f08ece87056e
```

Comment:
@rkazants , @ilya-lavrenov  could you please take a look at this change?
This fix doesn't solve caching problem, because we install this dependency using requirements file and we will build it again even it was saved in the cache. 
w/a to clone this repo manually and install this package then.
I don't think that is important now (wheel build takes 2 minutes)

Main question to you, do we except to pickup the newest commits in pre-commit tests?

Reply:
@mvafin, please take a look

Reply:
it is fine for now, would be good to update it regularly